cli/src/program.rs

```diff
@@ -356,6 +362,26 @@ impl ProgramSubCommands for App<'_, '_> {
                                 .help("/path/to/program.so"),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("dump-owned-accounts")
+                        .about("Write the program owned accounts to a directory in a format compatible with solana_test_validator")
```

Comment:
nit: maybe specify the command name to make things easier? Unless you mean with the crate?
```suggestion
                        .about("Write the program owned accounts to a directory in a format compatible with `solana-test-validator --account-dir`")
```